plugin-server/src/worker/vm/vm.ts

```diff
@@ -44,7 +55,16 @@ export function createPluginConfigVM(
         })
     }
 
-    const transformedCode = transformCode(indexJs, hub, AVAILABLE_IMPORTS)
+    const usedImports: Set<string> = new Set()
+    const transformedCode = transformCode(indexJs, hub, AVAILABLE_IMPORTS, usedImports)
+
+    const usedDeprecatedImports = [...usedImports].filter((i) => DEPRECATED_IMPORTS.has(i))
+    if (usedDeprecatedImports.length > 0) {
+        status.warn('⚠️', 'Plugin used deprecated import(s)', {
+            imports: usedDeprecatedImports,
+            pluginConfig: pluginConfig.id,
+        })
+    }
```

Comment:
Instead of this I'd love if we tracked all import usage and not by pluginConfig, but by pluginId as a prometheus metric.

Reply:
OK, changed to add a metric.

It may look a little weird since we compile source per VM (`PluginConfig`), but we don't want that level of cardinality, so I only add a gauge once per `Plugin`.

Should let us view import usage in the way we want, though.